hazelcast-jet-core/src/test/java/com/hazelcast/jet/core/ObservableResultsTest.java

```diff
@@ -553,15 +553,19 @@ public void unnamedObservable() {
     }
 
     @Test
-    public void getObservables() {
+    public void onlyObservedObservablesGetActivated() {
         //when
-        getObservable("a").addObserver(Observer.of(ConsumerEx.noop()));
-        getObservable("b");
-        getObservable("c").addObserver(Observer.of(ConsumerEx.noop()));
+        Observable<Object> a = jet().newObservable();
+        Observable<Object> b = jet().newObservable();
+        Observable<Object> c = jet().newObservable();
+
+        a.addObserver(Observer.of(ConsumerEx.noop()));
+        c.addObserver(Observer.of(ConsumerEx.noop()));
 
         //then
         Set<String> activeObservables = jet().getObservables().stream().map(Observable::name).collect(Collectors.toSet());
-        assertTrue(activeObservables.containsAll(Arrays.asList("a", "c")));
+        assertTrue(activeObservables.containsAll(Arrays.asList(a.name(), c.name())));
+        assertFalse(activeObservables.contains(b.name()));
```

Comment:
We can do this:
```suggestion
        assertEquals(new HashSet<>(Arrays.asList(a.name(), c.name())), activeObservables);
```
This way we'll see in the exception message what's the difference.

Reply:
Not really, because they are not equal (there are other, extra names in there too). Plus it wouldn't really help. All it would tell us is if `a` or `c` is missing, but that wouldn't help. Anyways, I'll investigate further if it happens again.